_includes/examples/scatterplot.svg

```diff
@@ -1,27 +1,88 @@
+<!-- This creates a svg area, if you draw outside
+     of these dimensions (350x160), it will not be
+     shown.
+   -->
 <svg width="350" height="160">
-  <!-- 60px x 10px margin -->
+  <!-- Remember, SVG uses 0,0 in the top left corner
+       of the drawable area, so push it with `translate`
+       right 60px and down 10px to create a nice margin.
+    -->
   <g class="layer" transform="translate(60,10)">
-    <!-- cx = 270px * ($X / 3)
-                ^      ^   ^
-    width of graph  x-value max(x)
+    <!-- We will now plot circles (size 5 radius dots
+         on the chart) in the svg area. We use some math
+         to figure out where to position them.
 
-          cy = 120px - (($Y / 80) * 120px)
-                 ^       ^     ^       ^
-      top of graph   y-value  max(y)  scale -->
+         Translation logic:
+
+         For the x-axis of each dot to plot: move cx over
+         by the porportional amount of 270 to fit the 4 data
+         points.
+
+         cx = 270px   *   ($X  /  3)
+                ^[1]        ^[2]  ^[3]
+
+         [1] width of graph = we choose 270 since it fits
```

Comment:
This is super verbose. I know the current thing is really terse, but the goal is that it fits next to the graph pretty well. Is there any particular part that you feel needs more clarification, or do you think all of this is necessary?